qiskit/providers/ibmq/ibmqbackend.py

```diff
@@ -181,15 +189,24 @@ def jobs(self, limit=50, skip=0, status=None, db_filter=None):
                                                   filter=api_filter)
         job_list = []
         for job_info in job_info_list:
+            kwargs = {}
             try:
-                ApiJobKind(job_info.get('kind', None))
+                job_kind = ApiJobKind(job_info.get('kind', None))
```

Comment:
I would put **only** the line `job_kind = ...` inside the `try` to highlight the fact that is at that point that it can fail. If you want to keep all the code inside the `try`, I'd add a comment clarifying the line producing the `ValueError`.

Reply:
Fixed at a80ce4c